scrapperFunctions/index.js

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable indent */
 'use strict';
+
+const ROWS_FROM_TABLE = require('../utils/getRows');
+const TABLE_TO_ARRAY = require('../utils/tableToArray');
```

Comment:
Not necessary need to be in UPERCASE

Reply:
Can extract only de funtion like `const { getRows } = require('../utils/getRows');`